### DISCUSSION TITLE
Found a way to easily write multiline comments, is this going to hit performance?

### DISCUSSION THREAD

**Discussion by nickheyer** [sagiegurari/duckscript#443] [Q&A]
![image](https://github.com/user-attachments/assets/bc2e30ee-3eda-4cf6-86fb-82aeb78b52dd)

Originally, I was going to do the python string method, but multi-line strings don't seem to be available. Nor is an unassigned string allowed (ideally it would be nice to see both).

**Comment by nickheyer** [comment]
Something like this would be nice. Even better if we can assign to a variable. 

![image](https://github.com/user-attachments/assets/927bf9c8-c2d2-4fb2-8029-fe80d3e95832)

**Comment by sagiegurari** [comment]
care to open an issue on that?
personally i'm for it, and did start investigating but never when for it.
